frontend/src/app/modules/boards/board/board-list/board-list-menu.component.html

```diff
@@ -0,0 +1,5 @@
+<ng-container class="board-list--more-menu" *ngIf="canDelete()">
```

Comment:
The `ngIf` is too restrictive. In cases where a user is not allowed to manage boards, but to edit versions, the menu is not shown.

Reply:
The comment is valid but as I only [copied over that part of the code](https://github.com/opf/openproject/pull/7397/files#diff-f6b9c062cae11ac21c750a25af8a4c9fL28) I am not going to change that situation. Currently it should not matter anyway, as it maps to the same permission in the backend.

Reply:
ok 👍